src/dlgConnectionProfiles.cpp

```diff
@@ -945,6 +945,9 @@ void dlgConnectionProfiles::slot_itemClicked(QListWidgetItem* pItem)
     const QString profileLoadedMessage = tr("This profile is currently loaded - close it before changing the connection parameters.");
 
     if (mudlet::self()->getHostManager().getHost(profile_name)) {
+        remove_profile_button->setEnabled(false);
+        remove_profile_button->setToolTip(utils::richText(tr("A profile that is in use cannot be removed!")));
```

Comment:
No need to shout at the user,

```suggestion
        remove_profile_button->setToolTip(utils::richText(tr("A profile that is in use cannot be removed")));
```

Tooltips are not meant to have periods at the end of them either (look at tooltips in the browser you're using an an example)

Reply:
Um, so what is that I see at the end of that sentence?
![image](https://github.com/user-attachments/assets/4e3d5969-a6b6-4bdb-8a07-38829c92294b)

Reply:
Huh, then I am wrong about the period.

Reply:
:smirk: I'm also wrong sometimes - don't let it get to you.

TBH I suspect there may be styling factors - but omitting the punctuation at the end of a tool-tip just feels wrong - especially when the tip is longer than just one short sentence (and without the period it wouldn't be a sentence) as then you'd have one (or other terminal punctuation mark like a '!' or a '?') at the end of the first but not one at the end of the last one.

Also, from a translation/localisation point of view, IMHO, one ideally should be thinking in terms of whole sentences - even short ones - as some languages (Spanish is the one that comes to mind) put punctuation at the beginning of a phrase or sentence as well as the end.

Reply:
As for the `!` https://en.wikipedia.org/wiki/Exclamation_mark#Languages suggests it is also used for the imperative or for emphasis - "You cannot do that!"

Reply:
Not in modern UI design though.